packages/bc-wallet-traction-adapter/src/util/CypherUtil.ts

```
@@ -0,0 +1,235 @@
+import bs58 from 'bs58'
+import { Buffer } from 'buffer'
+import crypto from 'crypto'
+
+import { environment } from '../environment'
+
+// TODO move to common package to dedup
+
+const env = environment.encryption
+
+function isChaCha20Poly1305Supported(): boolean {
+  try {
+    const supportedAlgorithms = crypto.getCiphers()
```

should it not only catch this line and leave the includes out of it, now we are eating all errors and returning false. maybe some other error happened?

---

Better to not catch anything then